compile/fullCompile/makeCompileArgsFull.m

```diff
@@ -56,8 +56,8 @@
 ARGS_1_1.numberOfDomainContrasts = coder.typeof(0);
 ARG = coder.typeof(0,[1 maxArraySize],[1 1]);
 ARGS_1_1.domainContrastLayers = coder.typeof({ARG}, [1 maxArraySize],[0 1]);
-ARGS_1_1.fitParams = coder.typeof(0,[maxArraySize 1],[1 0]);
-ARGS_1_1.otherParams = coder.typeof(0,[maxArraySize 1],[1 0]);
+ARGS_1_1.fitParams = coder.typeof(0,[1 maxArraySize],[0 1]);
```

Comment:
The PR description said "refactor the code, including packParams, to use row arrays" but this is a column array ?

Reply:
Ok, my understanding is this array has a single row, and multiple columns, something like [0, 0, 0]. Does that match what you think?

Reply:
You are right, mix up in head